docs/errors.md

```diff
@@ -872,7 +872,7 @@ Failed to query BlockHashFromNumber from graph node
 
 **Solution**
 
-Graph-node could not find the block hash given network and block number, check if graph-node has access to a network client that has synced to the required block. 
+Graph-node could not find the block hash given network and block number, check if graph-node has access to a network client that has synced to the required block, and that the network is configured in the default index-node.
```

Comment:
What is meant by "default index-node" here?

Reply:
updating this to be `the same index-node`? 
It seems that when indexers query blockHashFromNumber, the network had to be on the same index node, but I guess there's no `default`

Reply:
Sorry this still isn't crystal clear to me. Same index node as what?

Reply:
Do we mean, ensure that index node is configured with an RPC endpoint from the given network?